### ISSUE TITLE
influxdb won't start after headless transition

### ISSUE THREAD

**Issue by marclefevre** [influxdata/influxdb#13788]
I have a lubuntu system (18.10) running on PC hardware.  I got Influxdb up and running with no issues.  I then migrated to the system to headless to reduce power consumption.  After the move, influxdb will not start.

__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Change run level from graphical.target to multi-user.target
2. Shutdown and remove video card.
3. Restart system and remotely log in with ssh.

__Expected behavior:__
Expected influxdb to start successfully

__Actual behavior:__
Influxdb won't start:
```
May 04 16:17:37 automation systemd[1]: Started InfluxDB is an open-source, distributed, time series database.
May 04 16:17:38 automation influxd[2412]: ts=2019-05-04T21:17:38.113761Z lvl=info msg="InfluxDB starting" log_id=0FDBC9lG000 version=1.7.4 branch=1.7 commit=ef77e72f435b71b1ad6da7d6a6a4c4a262439379
May 04 16:17:38 automation influxd[2412]: ts=2019-05-04T21:17:38.113870Z lvl=info msg="Go runtime" log_id=0FDBC9lG000 version=go1.11 maxprocs=6
May 04 16:17:38 automation influxd[2412]: ts=2019-05-04T21:17:38.215009Z lvl=info msg="Using data dir" log_id=0FDBC9lG000 service=store path=/media/database/influxdb/data
May 04 16:17:38 automation influxd[2412]: ts=2019-05-04T21:17:38.215066Z lvl=info msg="Compaction settings" log_id=0FDBC9lG000 service=store max_concurrent_compactions=3 throughput_bytes_per_second=50331648 throughput_bytes_per_second_burst=50331648
May 04 16:17:38 automation influxd[2412]: ts=2019-05-04T21:17:38.215231Z lvl=info msg="Open store (start)" log_id=0FDBC9lG000 service=store trace_id=0FDBCA9l000 op_name=tsdb_open op_event=start
May 04 16:17:38 automation influxd[2412]: ts=2019-05-04T21:17:38.215376Z lvl=info msg="Open store (end)" log_id=0FDBC9lG000 service=store trace_id=0FDBCA9l000 op_name=tsdb_open op_event=end op_elapsed=0.150ms
May 04 16:17:38 automation influxd[2412]: run: open server: open tsdb store: mkdir /media/database/influxdb/data/_internal/_series: permission denied
May 04 16:17:38 automation systemd[1]: influxdb.service: Main process exited, code=exited, status=1/FAILURE
May 04 16:17:38 automation systemd[1]: influxdb.service: Failed with result 'exit-code'.
```

__Environment info:__

* Linux 4.18.0-17-generic x86_64
* InfluxDB v1.7.4 (git: 1.7 ef77e72f435b71b1ad6da7d6a6a4c4a262439379)

__Config:__
The storage location is the only changes to the configuration file:
```
[meta]
  # Where the metadata/raft database is stored
  dir = "/media/database/influxdb/meta"

  # Automatically create a default retention policy when creating a database.
  # retention-autocreate = true

  # If log messages are printed for the meta service
  # logging-enabled = true

###
### [data]
###
### Controls where the actual shard data for InfluxDB lives and how it is
### flushed from the WAL. "dir" may need to be changed to a suitable place
### for your system, but the WAL settings are an advanced configuration. The
### defaults should work for most systems.
###

[data]
  # The directory where the TSM storage engine stores TSM files.
  dir = "/media/database/influxdb/data"

  # The directory where the TSM storage engine stores WAL files.
  wal-dir = "/media/database/influxdb/wal"
```

The ownership of all of the files starting with /media is root:root and the permissions vary but I doubt that they changed as a result of the headless conversion.

__Logs:__
See above

__Performance:__
Generate profiles with the following commands for bugs related to performance, locking, out of memory (OOM), etc.

```sh
# Commands should be run when the bug is actively.
# Note: This command will run for at least 30 seconds.
curl -o profiles.tar.gz "http://localhost:8086/debug/pprof/all?cpu=true"
curl -o vars.txt "http://localhost:8086/debug/vars"
iostat -xd 1 30 > iostat.txt
# Attach the `profiles.tar.gz`, `vars.txt`, and `iostat.txt` output files.
```

**Comment by marclefevre** [comment]
I left out the fact that I removed an audio card at the same time I removed the video card.

Also, the directory "/media/database/influxdb/data/_internal/_series" already exists and has subdirectories and files below it.  Why would influxdb do a mkdir on it during startup?

**Comment by marclefevre** [comment]
I think I figured this out.  The database folder needs to have ownership of influxdb:influxdb since it is started for that user by systemd.  I can't explain why the ownership changed on my system or why it might have worked with the original root:root ownership.

As part of the troubleshooting, I tried to remove the database and recreate it but I couldn't do that when influxd wouldn't start.  So, I resorted to removing and reinstalling influxdb and found that I got the original permissions error again when recreating the database.  After looking again at the influxdb.service file in /etc/systemd/system, I found the reference to User=influxdb.
